node/src/Pos/Web/Server.hs

```diff
@@ -167,7 +168,7 @@ getUtxo :: HasConfiguration => WebMode ssc [TxOut]
 getUtxo = map toaOut . toList <$> GS.getAllPotentiallyHugeUtxo
 
 getLocalTxsNum :: WebMode ssc Word
-getLocalTxsNum = fromIntegral . length <$> getLocalTxs
+getLocalTxsNum = fromIntegral . length . getLocalTxs <$> getMemPoolSnapshot
```

Comment:
Is this a util or a top level web API handler? If it's top level then `getMemPoolSnapshot` is right, but if it's a util we should pass it in.

Reply:
Good question, sounds like I should check and pass it along if needed. Alas the fact every function runs in a generic (effect-bound) Monad `m` rather than a Servant Handler of sort means is harder for a programmer to know his boundaries 😉